src/main/webapp/styleComments.css

```diff
@@ -0,0 +1,58 @@
+input[type="number"],
```

Comment:
Be aware that general tags like this will start to conflict if anyone adds another stylesheet to your page. Fine for now though.